### ISSUE TITLE
Port dependency management from Ivy to Maven

### ISSUE THREAD

**Issue by andrus** [dflib/jjava#23]
The state of Ivy is concerning. See the discussion in https://github.com/SpencerPark/IJava/issues/121  Specifically https://lists.apache.org/thread/h372vt1ztd6gfmgmfkmqzrksx6fpw97g Let's see what it takes to port the kernel dependency resolving to Maven.

**Comment by maxandersen** [comment]
i would recommend you look into using mima (https://github.com/maveniverse/mima) its what I use for jbang as it is much nicer and simpler to "bootup" and have consistent behavior from.

**Comment by andrus** [comment]
@maxandersen , thanks for the advice. Will check it out. Using Maven directly proved to be somewhat of a pain in our experiments. It was never intended to be embedded.

**Comment by maxandersen** [comment]
yes, thats what mima helps on - also maven 3.9+ makes it much easier.

**Comment by stariy95** [comment]
Implemented with PR #61
